File: app/src/main/java/tw/com/louis383/coffeefinder/BaseFragment.java
package tw.com.louis383.coffeefinder;


import java.util.List;

import androidx.fragment.app.Fragment;
import tw.com.louis383.coffeefinder.model.entity.Shop;

/**
 * Created by louis383 on 2017/2/26.
 */

public abstract class BaseFragment extends Fragment {
    abstract public void prepareCoffeeShops(List<Shop> coffeeShops);
}

File: app/src/main/java/tw/com/louis383/coffeefinder/BaseFragment.kt
package tw.com.louis383.coffeefinder

import androidx.fragment.app.Fragment
import tw.com.louis383.coffeefinder.core.domain.model.CoffeeShop

/**
 * Created by louis383 on 2017/2/26.
 */
abstract class BaseFragment : Fragment() {
    abstract fun prepareCoffeeShops(coffeeShops: List<CoffeeShop>)
}